django_mailbox/models.py

```diff
@@ -372,6 +382,11 @@ def get_new_mail(self, condition=None):
         for message in connection.get_message(condition):
             msg = self.process_incoming_message(message)
             new_mail.append(msg)
+        self.last_polling = now()
+        if django.VERSION >= (1, 5):  # Django 1.5 introduces update_fields
+            self.save(update_fields=['last_polling'])
```

Comment:
This table only has a couple columns, it's *probably* not worth it to tell `save` which fields you've changed, but if you think otherwise, it's OK with me :-).

There's always a little bit of a balance between logical complexity and performance, and I'd rather we side with simplicity unless there are numbers to suggest we should do otherwise.